src/libaktualizr-posix/ipuptanesecondary.h

```diff
@@ -10,26 +10,25 @@ namespace Uptane {
 
 class IpUptaneSecondary : public SecondaryInterface {
  public:
-  static std::pair<bool, std::shared_ptr<Uptane::SecondaryInterface>> connectAndCreate(const std::string& address,
-                                                                                       unsigned short port);
+  static SecondaryInterface::Ptr connectAndCreate(const std::string& address, unsigned short port);
 
-  static std::pair<bool, std::shared_ptr<Uptane::SecondaryInterface>> create(const std::string& address,
-                                                                             unsigned short port, int con_fd);
+  static SecondaryInterface::Ptr create(const std::string& address, unsigned short port, int con_fd);
 
   explicit IpUptaneSecondary(const std::string& address, unsigned short port, EcuSerial serial,
                              HardwareIdentifier hw_id, PublicKey pub_key);
 
   // It looks more natural to return const EcuSerial& and const Uptane::HardwareIdentifier&
   // and they should be 'const' methods
```

Comment:
They are `const` methods now so that comment can be updated.

Reply:
yes.